src/OrleansAzureUtils/Providers/Streams/AzureQueue/IAzureQueueDataAdapter.cs

```diff
@@ -17,12 +17,12 @@ public interface IAzureQueueDataAdapter
         /// <summary>
         /// Creates a cloud queue message from stream event data.
         /// </summary>
-        CloudQueueMessage ToCloudQueueMessage<T>(Guid streamGuid, string streamNamespace, IEnumerable<T> events, Dictionary<string, object> requestContext);
+        CloudQueueMessage ToCloudQueueMessage<T>(SerializationManager serializationManager, Guid streamGuid, string streamNamespace, IEnumerable<T> events, Dictionary<string, object> requestContext);
 
```

Comment:
Similar to what you did with other streaming parts, can this be avoided and instead make it a constructor dependency of the concrete instances (and also remove the new() constraint)?